-all not working

there is a bug when i use -all arg...

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x7ac576]

goroutine 63 [running]:
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.proccesItems(0xa15d00, 0xc00008e060, 0xc000330000, 0x64, 0x80, 0xc0002480a0, 0x95dbdb, 0x6, 0xc00008a000, 0xc000200780, ...)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:145 +0x226
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a92c, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:112 +0x530
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a76c, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a22c, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a5ac, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a3ec, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00016c06c, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc00032a06c, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc0001281c1, 0x5b, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).enumerate(0xd96fd8, 0xa15d00, 0xc00008e060, 0xc0002007e0, 0x54, 0xc0002480a0, 0xc0002101e0, 0xc00008a000, 0xc000200780)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:128 +0x6d3
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).Run.func1(0xc000200780, 0xc00008a000, 0xc000018100, 0xb, 0xd96fd8, 0xa15d00, 0xc00008e060)
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:55 +0x1d1
created by github.com/projectdiscovery/subfinder/pkg/subscraping/sources/github.(*Source).Run
        /home/runner/work/subfinder/subfinder/pkg/subscraping/sources/github/github.go:45 +0xa5

**Screenshots**
![image](https://user-images.githubusercontent.com/70751674/92372815-0f8f5f80-f130-11ea-9e85-8ea8e5e8ee2a.png)

--- a/pkg/subscraping/sources/github/github.go
+++ b/pkg/subscraping/sources/github/github.go
@@ -138,8 +138,8 @@ func proccesItems(ctx context.Context, items []item, domainRegexp *regexp.Regexp
 		if err != nil {
 			if resp != nil && resp.StatusCode != http.StatusNotFound {
 				session.DiscardHTTPResponse(resp)
-				return err
 			}
+			return err
 		}
 
 		if resp.StatusCode == http.StatusOK {

